.github/sync.yml

```
@@ -36,6 +36,26 @@ group:
       neurobagel/api
       neurobagel/bagel-cli
       neurobagel/federation-api
+  - files:
```

very minor 🍒, to keep spacing consistent in the file hehe
```suggestion

  - files:
```